docs/api/IoT.js-API-BLE.md

```diff
@@ -104,4 +84,97 @@ ble.on('advertisingStart', function(error) {
     ]);
   }
 });
-```
\ No newline at end of file
+```
+
+
+## Class: Descriptor
+
+### new Descriptor(options)
+
+* `options` {Object}
+  * `uuid:` {string}
+  * `value` {string|Buffer}
+
+**Example**
+
+```js
+var descriptor = new Descriptor({
+    uuid: '2901',
+    value: 'value'
+});
+```
+
+
+## Class: Characteristic
+
+### new Characteristic(options)
+
+* `options` {Object}
+  * `uuid:` {string}
+  * `properties` {[string]} can be a combination of 'read', 'write', 'writeWithoutResponse', 'notify', 'indicate'
+  * `secure` {[string]} enables security for properties, can be a combination of 'read', 'write', 'writeWithoutResponse', 'notify', 'indicate'
+  * `value` {Buffer}
+  * `descriptors` {[Descriptor]}
+  * `onReadRequest` {Function} optional read request handler
+    * `offset` {number} (0x0000 - 0xffff)
+    * `callback` {Function}
+      * `result` {Characteristic.RESULT_*}
+      * `data` {Buffer}
+  * `onWriteRequest` {Function} optional write request handler
+    * `data` {Buffer}
+    * `offset` {number} (0x0000 - 0xffff)
+    * `withoutResponse` {boolean}
+    * `callback` {Function}
+      * `result` {Characteristic.RESULT_*}
+  * `onSubscribe` {Function} optional notify/indicate subscribe handler
+    * `maxValueSize` {number} maximum data size
+    * `updateValueCallback` {Function}
+  * `onUnsubscribe` {Function} optional notify/indicate unsubscribe handler
+  * `onNotify` {Function} optional notify sent handler
+  * `onIndicate` {Function} optional indicate confirmation received handler
```

Comment:
I'm still confused. There is a description. `optional indicate confirmation received handler`